src/components/NavigationBar/NavigationBar.tsx

```diff
@@ -76,7 +76,7 @@ const NavigationBar = () => {
 
 	return (
 		<nav
-			className="fixed top-0 w-full z-50 opacity-80 bg-black
```

Comment:
This change is already fixed in another PR, other than that everything is fine!

Reply:
would you happen to know if these lint errors were left in intentionally? am I good to push up this commit (which is taking latest main) with no verify or should we look to fix these up?

![Screenshot 2024-10-03 175032](https://github.com/user-attachments/assets/9c48c512-ad47-4cc6-b2bd-486f70bd5a69)

Reply:
Thanks @wlee261 for the PR and @HusseinSerag for helping with the review 😊